pkg/test/kds/samples/resources.go

```diff
@@ -241,7 +241,7 @@ var (
 			Match: map[string]string{"serivce": "*"},
 		}},
 		Conf: &mesh_proto.ProxyTemplate_Conf{
-			Imports: []string{"default-kuma-profile"},
+			Imports: []string{"default-proxy"},
```

Comment:
Why this change? Does it somehow affect the tests?

Reply:
yes, it is throwing a validation error on the inspect dataplane test suite